app/src/main/java/com/framgia/englishconversation/screen/timeline/TimelineAdapter.java

```diff
@@ -1,23 +1,36 @@
 package com.framgia.englishconversation.screen.timeline;
 
+import android.net.Uri;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
```

Comment:
ERROR: Unused import: android.support.annotation.Nullable. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports